Source/NetworkReachabilityManager.swift

```diff
@@ -195,9 +201,11 @@ open class NetworkReachabilityManager {
     open func stopListening() {
         SCNetworkReachabilitySetCallback(reachability, nil, nil)
         SCNetworkReachabilitySetDispatchQueue(reachability, nil)
```

Comment:
Newline?

Reply:
These are all statements to reset some state, so a separation doesn't seem necessary.

Reply:
Ah no worries, I generally always put newlines in front of conditionals.